examples/SEA076_20230906T0852_nrt.cdl

```diff
@@ -0,0 +1,389 @@
+netcdf SEA076_20230906T0852_nrt {
+dimensions:
+	N_MEASUREMENTS = 10 ;
+	string24 = 24 ;
+	string7 = 7 ;
+	string11 = 11 ;
+variables:
+	float CNDC(N_MEASUREMENTS) ;
+		CNDC:_FillValue = NaNf ;
+		CNDC:instrument = "instrument_ctd" ;
+		CNDC:long_name = "Electrical conductivity of the water body by CTD" ;
+		CNDC:observation_type = "measured" ;
+		CNDC:standard_name = "sea_water_electrical_conductivity" ;
+		CNDC:units = "mS cm-1" ;
+		CNDC:valid_max = 85. ;
+		CNDC:valid_min = 0. ;
+		CNDC:URI = "https://vocab.nerc.ac.uk/collection/OG1/current/CNDC/" ;
+		CNDC:ancillary_variables = "CNDC_QC" ;
+		CNDC:coordinates = "DEPTH LATITUDE LONGITUDE TIME" ;
+	int64 CNDC_QC(N_MEASUREMENTS) ;
+		CNDC_QC:ioos_qc_module = "qartod" ;
+		CNDC_QC:quality_control_conventions = "IOOS QARTOD standard flags" ;
+		CNDC_QC:quality_control_set = 1LL ;
+		CNDC_QC:valid_min = 1LL ;
+		CNDC_QC:valid_max = 9LL ;
+		CNDC_QC:flag_values = 1LL, 2LL, 3LL, 4LL, 9LL ;
+		CNDC_QC:flag_meanings = "GOOD UNKNOWN SUSPECT FAIL MISSING" ;
+		CNDC_QC:long_name = "Electrical conductivity of the water body by CTD Quality Flag" ;
+		CNDC_QC:standard_name = "status_flag" ;
+		CNDC_QC:comment = "Quality control flags from IOOS QC QARTOD https://github.com/ioos/ioos_qc Version: 2.1.0. Using config: [<Call stream_id=conductivity function=qartod.gross_range_test(suspect_span=[6, 42], fail_span=[3, 45])>, <Call stream_id=conductivity function=qartod.location_test(bbox=[10, 50, 25, 60])>].  Threshold values from EuroGOOS DATA-MEQ Working Group (2010) Recommendations for in-situ data Near Real Time Quality Control [Version 1.2]. EuroGOOS, 23pp. DOI http://dx.doi.org/10.25607/OBP-214." ;
+		CNDC_QC:coordinates = "DEPTH LATITUDE LONGITUDE TIME" ;
+	float PHASE(N_MEASUREMENTS) ;
```

Comment:
Maybe use an integer type here? Probably `int8` would be enough?

Reply:
I believe int8 is functionally the same as byte here I believe. No need to change the doc Victor